pkgs/by-name/ma/material-maker/package.nix

```
@@ -0,0 +1,140 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  autoPatchelfHook,
+  copyDesktopItems,
+  makeDesktopItem,
+  godot3-export-templates,
+  godot3-headless,
+  alsa-lib,
+  libGL,
+  libGLU,
+  libpulseaudio,
+  libX11,
+  libXcursor,
+  libXext,
+  libXfixes,
+  libXi,
+  libXinerama,
+  libXrandr,
+  libXrender,
+  zlib,
+  udev,
+}:
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

See https://github.com/NixOS/nixpkgs/issues/315337#issuecomment-2138814370